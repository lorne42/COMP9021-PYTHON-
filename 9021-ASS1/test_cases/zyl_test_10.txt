Yesterday, I was playing League of Legends. 
Sir Darius said: "At least one of us is a Knight."
Sir Garen said: "At most one of Sir Jinx, Sir Lee or Sir Teemo is a Knight!"
Sir Jinx said: "exactly one of Sir Garen, Sir Darius and Sir Riven is a Knave."
Sir Lee said: "All of us are Knights."
Sir Riven said: "I am a Knight."
Sir Teemo said, "Sir Darius is a Knight."
Sir Darius said: "Sir Garen or Sir Jinx is a Knight."
Sir Lee agreed: "Sir Garen and Sir Riven are Knaves."
What do you think? 