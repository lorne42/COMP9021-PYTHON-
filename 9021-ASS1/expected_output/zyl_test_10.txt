The Sirs are: Darius Garen Jinx Lee Riven Teemo
There are 2 solutions.